livy/src/test/java/org/apache/zeppelin/livy/LivyInterpreterIT.java

```diff
@@ -763,6 +763,39 @@ public void testLivyTutorialNote() throws IOException {
     }
   }
 
+  @Test
+  public void testLivyTracebackMsg() throws LivyException {
```

Comment:
Could you merge this test into other tests ? Because each test need to create a new livy session which would lots of time, merging into other test could help reduce test time.